packages/terra-i18n/docs/UPGRADEGUIDE.md

```diff
@@ -1,5 +1,36 @@
 # Terra i18n Upgrade Guide
 
+## Changes from 3.0 to 4.0
+
+The terra-i18n `aggregate-translations` pre-build tool has been removed from this package.
+
+Along with this, the post-install script which wrote default intl and translation loaders has been removed.
+
+Use the [`aggregate-translations` pre-build tool from terra-toolkit instead.](https://github.com/cerner/terra-toolkit/blob/master/docs/AggregateTranslations.md)
+
+If you require translations in your Jest tests, create a `jestglobalsetup.js` file in your app.
```

Comment:
I'm not sure if the jest setup information is relevant here. It seems like this should be included in the terra-ui testing i18n components page.

Reply:
That is a good point, I'll move this over to the terra-ui doc site.

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2288/commits/6245e266ef54ae055263c4b8679b3c2dca060b12

I've started a new branch in terra-ui with this info. I'll send out a PR for that later on once terra-enzyme-intl is released as I want to include that info in the terra-ui i18n testing doc update.

Reply:
Here is a the PR for moving this to terra-ui doc site: https://github.com/cerner/terra-ui/pull/158